fiber/adapter.go

```diff
@@ -129,8 +132,16 @@ func (f *FiberLambda) adaptor(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	remoteAddr, err := net.ResolveTCPAddr("tcp", r.RemoteAddr)
+	// We need to make sure the net.ResolveTCPAddr call works as it expects a port
+	addrWithPort := r.RemoteAddr
+	if !strings.Contains(r.RemoteAddr, ":") {
+		addrWithPort = r.RemoteAddr + ":80" // assuming a default port
```

Comment:
Should we be hard coding this port? Regardless, shouldn't we assume port 443 instead for https?

Reply:
Do we know why this net.ResolveTCPAddr is even done?

Reply:
Fiber uses fasthttp and fasthttp requires a ctx and the ctx requires a *[TCPAddr](https://pkg.go.dev/net#TCPAddr).

Reply:
Right. Before the change you mentioned that broke this. It was indeed set to nil. It got set to a value in a change that added ALB support.

Pretty sure we can just this to nil again here, no?

Reply:
No. Then you don't have the source ip in the logs.